ansible_collections/arista/avd/roles/eos_designs/python_modules/l3_edge/utils.py

```diff
@@ -166,12 +166,13 @@ def _resolve_p2p_ips(self, p2p_link: dict) -> dict:
 
         else:
             # Resolving subnet from pool
-            ip_pool = get_item(self._p2p_links_ip_pools, "name", p2p_link["ip_pool"], default={}).get("ipv4_pool")
-            if not ip_pool:
-                # Not possible to resolve from pool. Returning original
+            ip_pool = get_item(self._p2p_links_ip_pools, "name", p2p_link["ip_pool"], default={})
+            ip_pool_subnet = ip_pool.get("ipv4_pool")
+            if not ip_pool_subnet:
                 return p2p_link
+            pool_prefix = int(ip_pool.get("prefix_size", 31))
```

Comment:
please call the var `prefix_size` to make it simpler to follow.

Reply:
done